src/engine.tsx

```diff
@@ -331,7 +331,7 @@ const Engine: {
       // Bladeburner offline progress
       if (Player.bladeburner) Player.bladeburner.storeCycles(numCyclesOffline);
 
-      Go.storeCycles(numCyclesOffline);
+      Go.storeCycles(numCyclesOffline / 3);
```

Comment:
This seems maybe too harsh? In any case, this should be applied inside `storeCycles`, instead of in engine, for cleanliness.